src/status_im2/contexts/onboarding/common/syncing/render_device.cljs

```diff
@@ -0,0 +1,56 @@
+(ns status-im2.contexts.onboarding.common.syncing.render-device
```

Comment:
May I ask why the verb `render` is used in the namespace? The namespace is not a function to be called, so it's kind of confusing (at least to me). Could you name it just `...common.syncing.device`? Or even create a directory named `device` and inside a file named `view.cljs`?